zulipterminal/config/keys.py

```diff
@@ -385,6 +385,11 @@ class KeyBinding(TypedDict):
         'help_text': 'Clear compose box',
         'key_category': 'msg_compose',
     },
+    'OPEN_EXTERNAL_EDITOR': {
+        'keys': ['ctrl o'],
+        'help_text': 'Open the message in external editor',
```

Comment:
I like the change to 'external editor' rather than 'terminal editor' here, particularly if we're working with external editors more generally. I did try gvim, but it didn't seem to work for me, though vim still does.

'Open' vs 'Edit' is tricky; maybe use 'Compose'?

A minor point is that this may make more sense to have further up in the file, so that in the help and hotkeys.md it appears above the 'navigation' compose keys (eg. "Jump to start of line")

Reply:
I think using the first letter of the command as the shortcut is easier to remember if 'Open' keyword is okay in this context.

Reply:
I agree about 'o' for 'Open' being good, I'll have a think on the wording.